packages/system/src/globals/global-constants.tsx

```diff
@@ -33,6 +34,15 @@ export const FRAMEWORK_RESOURCES = (bigSize = false): LinkCardProps[] => [
 		bigSize,
 	},
 	{
+		title: "SolidJS Resources",
+		href: "https://solidjs.framework.dev/",
+		Icon(props) {
+			return <SolidJSIcon {...props} />
+		},
+		backgroundColor: "#2c4f7c",
+    bigSize,
```

Comment:
Weird indenting? Not a blocker, but would be nice to see fixed.

Reply:
fixed